sql/src/test/java/io/crate/execution/engine/distribution/merge/RamAccountingPageIteratorTest.java

```diff
@@ -104,17 +93,28 @@ public void testWrappingApplied() {
         assertThat(((RamAccountingPageIterator) pagingIterator1).delegatePagingIterator,
                    instanceOf(PassThroughPagingIterator.class));
 
-        PagingIterator<Integer, Row> pagingIterator2 = PagingIterator.create(
+        PagingIterator<Integer, Row> repeatingSortedPagingIterator = PagingIterator.create(
             2,
             true,
             PositionalOrderBy.of(
                 new OrderBy(Collections.singletonList(Literal.of(1)), new boolean[] {false}, new Boolean[] {false}),
                 Collections.singletonList(Literal.of(1))),
             () -> null);
 
-        assertThat(pagingIterator2, instanceOf(RamAccountingPageIterator.class));
-        assertThat(((RamAccountingPageIterator) pagingIterator2).delegatePagingIterator,
+        assertThat(repeatingSortedPagingIterator, instanceOf(RamAccountingPageIterator.class));
+        assertThat(((RamAccountingPageIterator) repeatingSortedPagingIterator).delegatePagingIterator,
                    instanceOf(SortedPagingIterator.class));
+
+        PagingIterator<Integer, Row> nonRepeatingSortedPagingIterator = PagingIterator.create(
```

Comment:
Would be nice if we could create a separate test case for this and make the test method names more meaningful